tree/ntuple/v7/test/ntuple_print.cxx

```diff
@@ -0,0 +1,93 @@
+#include <ROOT/RNTuple.hxx>
+#include <ROOT/RNTupleModel.hxx>
+#include <ROOT/RFieldVisitor.hxx>
+
+#include <TClass.h>
+#include <TFile.h>
+
+#include "gtest/gtest.h"
+
+#include <sstream>
+#include <iostream>
+
+using RNTupleReader = ROOT::Experimental::RNTupleReader;
+using RNTupleWriter = ROOT::Experimental::RNTupleWriter;
+using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RFieldBase = ROOT::Experimental::Detail::RFieldBase;
+using RPrintVisitor = ROOT::Experimental::RPrintVisitor;
+template <class T>
+using RField = ROOT::Experimental::RField<T>;
+
+namespace {
+    
+    /**
+     * An RAII wrapper around an open temporary file on disk. It cleans up the guarded file when the wrapper object
+     * goes out of scope.
+     */
+    class FileRaii {
+    private:
+        std::string fPath;
+    public:
+        FileRaii(const std::string &path) : fPath(path)
+        {
+        }
+        FileRaii(const FileRaii&) = delete;
+        FileRaii& operator=(const FileRaii&) = delete;
+        ~FileRaii() {
+            std::remove(fPath.c_str());
+        }
+    };
+    
+} // anonymous namespace
+
+TEST(RNtuplePrint, FullString)
+{
+    TFile *file = TFile::Open("test.root", "RECREATE");
+    FileRaii fileGuard("test.root");
```

Comment:
Could you maybe combine the RAII and the `TFile` construction? E.g. by adding a `FileRAII::GetFile()`? That way, `FileRAII` could close and delete, on all control paths.